tests/index/mongo_atlas/__init__.py

```diff
@@ -40,7 +39,7 @@ def assert_when_ready(callable: Callable, tries: int = 5, interval: float = 2):
         except AssertionError:
             tries -= 1
             if tries == 0:
-                raise
+                raise RuntimeError("Retries exhausted.")
```

Comment:
We are missing the original assertion message. Could we pass it through? Maybe we can Add retries exhausted + the assertion message?

Reply:
@WaVEV Ok. I've updated to raise from so the trace contains the assertion that failed.